tests/conftest.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 """Base fixtures for unit tests"""
 
-from mock import Mock
+from unittest.mock import Mock
```

Comment:
Two options here:
1. use standalone mock module as openstack does
2. use six
```python
import six
six.add_moves(six.MovedModule('mock', 'mock', 'unittest.mock')
from six.moves import mock
from mock import Mock 
```

Reply:
Ah, I'd be happy with 1) too, that'd resolve the problem I mentioned. I guess we never added 'mock' explicitly to the requirements.txt to begin with, that'd be enough.

Reply:
Let’s go with the direct import of mock since it works in all cases and does not require six.

setup.py does have mock in the test requirements. I would prefer keeping requirements.txt for only run time dependencies.

Reply:
PR#25 adds a test-requirements.txt file where I explicitly add mock module so it should be fine then.